app/src/main/java/com/p2p/presentation/countdown/CountdownActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.p2p.presentation.countdown
+
+import android.os.Bundle
+import android.view.View
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
+import android.widget.ImageView
+import com.p2p.R
+import com.p2p.presentation.base.BaseActivity
+
+
+class CountdownActivity : BaseActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_countdown)
+
+        val firstCountdownElement : ImageView = findViewById(R.id.countdown_3)
+        val secondCountdownElement : ImageView = findViewById(R.id.countdown_2)
+        val thirdCountdownElement : ImageView= findViewById(R.id.countdown_1)
+
+        val firstZoomInOutAnimation: Animation =
+            AnimationUtils.loadAnimation(applicationContext, R.anim.zoom_in_out)
+        val secondZoomInOutAnimation: Animation =
+            AnimationUtils.loadAnimation(applicationContext, R.anim.zoom_in_out)
+        val thirdZoomInOutAnimation: Animation =
+            AnimationUtils.loadAnimation(applicationContext, R.anim.zoom_in_out)
+
+        secondZoomInOutAnimation.startOffset = 1500
+        thirdZoomInOutAnimation.startOffset = 3000
+
+        val countdownElements: List<ImageView> =
```

Comment:
podrias las listas tenerlas directamente de los resources, y dsp haces el .map y le pones a todas por ejemplo


countdounResources = listof(R.id.countdown_1,R.id.countdown_2,R.id.countdown_3)
countDownResources.map(findViewById(it))

idem pa el otro

Reply:
Genial ahi lo modifico! Al otro no puedo, porque despues modifico unos atributos particulares para cada animacion